mm2src/mm2_bin_lib/src/lib.rs

```diff
@@ -108,3 +108,20 @@ async fn finalize_mm2_stop(ctx: MmArc) {
     dispatch_lp_event(ctx.clone(), StopCtxEvent.into()).await;
     let _ = ctx.stop().await;
 }
+
+#[cfg_attr(target_arch = "wasm32", derive(serde::Serialize))]
+#[derive(Clone, Debug, PartialEq, Primitive)]
+pub enum StartupErrorCode {
+    /// Operation completed successfully
+    Ok = 0,
```

Comment:
It seems better to call this `StartupResultCode` instead of `StartupErrorCode` as it also includes success code.

Reply:
i mean remove the Ok variant all together from this enum.
the `()` suggestion is about the other functions in which we return `Result<i8, Error>`. there is no point of returning `i8` in the success variant since it's already the `Ok` variant of the `Result` (unless we have different success types, which we currently don't).

Reply:
I understand that, what I am saying is that `0` is useful when converting things from other languages (which is the reason why we use this type).

Reply:
> the () suggestion is about the other functions in which we return Result<i8, Error>. there is no point of returning i8 in the success variant since it's already the Ok variant of the Result (unless we have different success types, which we currently don't).

I opted for returning 0 from wasm (javascript) function to allow GUIs to handle the success case the same way if possible. I would ask for @CharlVS opinion on this and which would be better for GUIs.

I will rename this to `StartupResultCode` for now

Reply:
done here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2389/commits/c26088dc1126e6f0aa8b0300c2943fbe395a5c0d

Reply:
If you want to keep this type compatible with FFI, you should keep `0` inside of it.